jni/com_eclipsesource_v8_V8Impl.h

```diff
@@ -501,6 +501,30 @@ JNIEXPORT void JNICALL Java_com_eclipsesource_v8_V8__1processDebugMessages
 
 /*
  * Class:     com_eclipsesource_v8_V8
```

Comment:
The comment at the top says this file was auto generated, but looking at the git history made it seem like maybe that was just the case in the past and is no longer true. If there's another way these header declarations should be generated, let me know.